authorization-service/.eslintignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+public
+serverless.yml
```

Comment:
We do not use this file, I think?

serverles.yml

Reply:
done